charts/trino/values.yaml

```diff
@@ -218,6 +218,13 @@ securityContext:
   runAsUser: 1000
   runAsGroup: 1000
 
+# -- SecurityContext configuration for containers
+containerSecurityContext:
+  allowPrivilegeEscalation: false  #  Controls whether a process can gain more privileges than its parent process.
+  capabilities:
+    drop:  # List of Linux kernel capabilities that are dropped from every container. You can confirm the options for "capabilities" here:　https://man7.org/linux/man-pages/man7/capabilities.7.html  Please make sure to remove "CAP_" prefix which the kernel attaches to the names of permissions.
```

Comment:
Use

List of Linux kernel capabilities that are dropped from every container. Valid valeus are listed at https://man7.org/linux/man-pages/man7/capabilities.7.html  Ensure to remove the "CAP_" prefix which the kernel attaches to the names of permissions.

Reply:
Can you adjust to this?

Reply:
oh, sorry. Applied your suggestion! Thank you!